vernac/loadpath.ml

```diff
@@ -252,28 +252,6 @@ let locate_qualified_library ?root qid :
       Ok (library, file)
     | Error _ as e -> e
 
-let error_unmapped_dir qid =
-  let prefix, _ = Libnames.repr_qualid qid in
-  CErrors.user_err
-    Pp.(seq [ str "Cannot load "; Libnames.pr_qualid qid; str ":"; spc ()
-            ; str "no physical path bound to"; spc ()
-            ; DP.print prefix; fnl ()
-            ])
-
-let error_lib_not_found qid =
-  let vos = !Flags.load_vos_libraries in
-  let vos_msg = if vos then [Pp.str " (while searching for a .vos file)"] else [] in
-  CErrors.user_err
-    Pp.(seq ([ str "Cannot find library "; Libnames.pr_qualid qid; str" in loadpath"]@vos_msg))
```

Comment:
Instead of moving this stuff to vernacinterp I'd rather add `raise : t -> 'a` in module Error in this file
or at least `pp : t -> Pp.t`

Reply:
Yes, added a pp function. This is still not very satisfying, as for example in `coq-lsp` such errors are not exactly printed that way, for example we have information from the workspace configuration so we can suggest a fix to the user.

It is not easy to figure what's the right API here until I progress a bit more on the `coq-lsp` side.

I propose that for now, we leave things as they are, but for sure we will need some other changes.

Reply:
Added a raise in the end.